src/python/pants/backend/project_info/tasks/export.py

```diff
@@ -271,9 +275,15 @@ def iter_transitive_jars(jar_lib):
 
       if isinstance(current_target, JvmTarget):
         info['excludes'] = [self._exclude_id(exclude) for exclude in current_target.excludes]
-        info['platform'] = current_target.platform.name
+        if current_target.platform:
+          info['platform'] = current_target.platform.name
+        else:
+          info['platform'] = self._default_jvm_platform_name
```

Comment:
Could save some lines and make intent clearer with 
```python
info['platform'] = current_target.platform.name if current_target.platform else self._default_jvm_platform_name
```

Reply:
Done!